com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -491,6 +491,8 @@ static class RenderPassBlock
 
         internal bool useDepthPriming { get; set; } = false;
 
+        internal bool stripShadowsOffVariants { get; set; } = false;
```

Comment:
Same question here as above, is this only relevant to shadows or can we remove that prefix?

Reply:
yeap only shadows and additional lights